lib/elasticband/aggregation.rb

```diff
@@ -93,6 +100,15 @@ def parse_top_hits(root_aggregation, options_top_hits)
         aggregation = Aggregation::TopHits.new(name, size, options.except(*PARSE_AGGREGATIONS))
         parse_aggregations(options, aggregation)
       end
+
+      def parse_filter(aggregation_name, options)
```

Comment:
I found confused because the method needs 2 parameters (name and options) and when you call it you use a splat in an array variable. I think that we need to improve the readability of the way we pass the parameters to this method. What do you think?